### PR TITLE
Lvnod 30 config database

### PR DESCRIPTION
**Pull request by shyyka**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)

## Code reviewers

- [x] @yarvanya
- [x] @yurii-kokhalevych
- [x] @avoytovich

### Second Level Review

- [x] @lhalam
- [ ] @4asof4ik

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo
## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: config/secretkey.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"facebook": "",
+	"google": ""
+}
```
Comment (lhalam) [inline config/secretkey.json]: add end line

File: config/secretkey.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+	"facebook": "",
```
Comment (lhalam) [inline config/secretkey.json]: use 2 spaces

File: server/app.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const logger = require('morgan');
+const bodyParser = require('body-parser');
+
+const app = express();
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+//require('./router')(app);
```
Comment (lhalam) [inline server/app.js]: remove comment

File: config/secretkey.json (hunk 1 above)
Comment (shyyka) [reply config/secretkey.json]: done

File: server/app.js (hunk 3 above)
Comment (shyyka) [reply server/app.js]: done

File: server/model/index.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(module.filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(`${__dirname}/../../config/db.json`)[env];
+const db = {};
+
+let sequelize;
+if (process.env.DATABASE_URL) {
+  sequelize = new Sequelize(process.env.DATABASE_URL,config);
```
Comment (AlbaMontis) [inline server/model/index.js:11]: Missed space between 'process.env.DATABASE_URL,config'

File: server/routes/index.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  app.get('/dami', (req, res) => res.status(200).send({
+    message: 'Welcome to the Da-Mi API!',
+  }));
+
+  app.post('/dami/users', usersController.create);
+  app.get('/dami/users', usersController.list);
+  app.get('/dami/users/:userId', usersController.retrieve);
+  app.put('/dami/users/:userId', usersController.update);
+  app.delete('dami/users/:userId', usersController.destroy);
+
```
Comment (AlbaMontis) [inline server/routes/index.js:14]: Useless blank line.

File: config/db.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": "donttry123",
+    "database": "dami-dev",
+    "host": "0.0.0.0",
+    "port": 5432,
+    "dialect": "postgres"
+  },
+  "production": {
```
Comment (omaik) [inline config/db.json]: Storing configurations for production environment directly in code is not secure

File: .sequelizerc [hunk 7]
```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+module.exports = {
+  "config": path.resolve('./config', 'db.json'),
+  "models-path": path.resolve('./server/models'),
+  "seeders-path": path.resolve('./server/seeders'),
+  "migrations-path": path.resolve('./server/migrations')
+};
```
Comment (4asof4ik) [inline .sequelizerc:8]: add helper

File: package.json [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  "dependencies": {
+    "body-parser": "^1.17.1",
+    "express": "^4.15.2",
+    "morgan": "^1.8.1",
+    "pg": "^3.4.3",
+    "pg-hstore": "^2.3.2",
+    "sequelize": "^3.12.2",
+    "webpack": "^2.5.1",
+    "webpack-dev-server": "^2.4.5"
```
Comment (4asof4ik) [inline package.json:17]: should be in devDepen

File: server.js [hunk 9]
```diff
@@ -0,0 +1,10 @@
+const http = require('http');
+const app = require('./server/app');
+
```
Comment (4asof4ik) [inline server.js:3]: use express

**Pull request closed without merging**